superset/models/dashboard.py

```diff
@@ -397,6 +427,11 @@ def get(cls, id_or_slug: str) -> Dashboard:
         qry = session.query(Dashboard).filter(id_or_slug_filter(id_or_slug))
         return qry.one_or_none()
 
+    def am_i_owner(self) -> bool:
+        if g.user is None or g.user.is_anonymous or not g.user.is_authenticated:
+            return False
+        return g.user.id in set(map(lambda user: user.id, self.owners))
```

Comment:
IMO `is_actor_owner()` would be a more appropriate name for this method.

Reply:
> IMO `is_actor_owner()` would be a more appropriate name for this method.

agree